dds_web/api/project.py

```diff
@@ -977,3 +977,49 @@ def get(self):
 
         return_info = {"project_info": project_info}
         return return_info
+
+    @auth.login_required(role=["Unit Admin", "Unit Personnel", "Project Owner"])
+    @logging_bind_request
+    @json_required
+    @handle_validation_errors
```

Comment:
I don't believe this is needed since we catch the Validation error, but there's a db_session decorator or what ever it's called that we can use
```suggestion
```